packages/kbn-plugin-helpers/tasks/build/__fixtures__/create_build_test_plugin/package.json

```diff
@@ -5,7 +5,7 @@
     "version": "6.0.0"
   },
   "dependencies": {
-
+    "noop": "0.2.0"
```

Comment:
Interesting choice. It really doesn't matter at all, but [noop3](https://www.npmjs.com/package/noop3) links to a git repo, has less code, and has a killer logo:

![](https://github.com/sindresorhus/noop3/raw/master/logo.png)

Reply:
lol, deal

Reply:
Also a killer version number

Reply:
And a robot shark!

![](https://cloud.githubusercontent.com/assets/170270/11731042/eba6ffc6-9f98-11e5-8d7d-7890dbc394c5.gif)